src/main/proto/fleet.proto

```diff
@@ -22,4 +22,10 @@ message CompilerFleetRequest {
     bytes zipBytes = 1;
     string rootFileName = 2;
     string taskId = 3;
```

Comment:
While we're not launched yet, let's swap the order, so that `taskId` had the same number (=1) in both request and result.

Reply:
Done